public/forms/builder/meta.ttl

```diff
@@ -1,125 +1,145 @@
-@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix ns1:  <http://lblod.data.gift/concept-schemes/> .
-@prefix skos:   <http://www.w3.org/2004/02/skos/core#> .
-ns1:c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6  rdf:type  skos:ConceptScheme .
-@prefix ns3:  <http://mu.semte.ch/vocabularies/core/> .
-ns1:c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6  ns3:uuid  "c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6" ;
-  skos:prefLabel  "Form input types"@en .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix core: <http://mu.semte.ch/vocabularies/core/> .
+@prefix form: <http://lblod.data.gift/vocabularies/forms/> .
+@prefix display: <http://lblod.data.gift/display-types/> .
 
-@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix skos:   <http://www.w3.org/2004/02/skos/core#> .
-@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
-
-<http://lblod.data.gift/display-types/defaultInput>
+display:defaultInput
   rdf:type  rdfs:Class , skos:Concept ;
-  <http://mu.semte.ch/vocabularies/ext/displayType>   "defaultInput" ;
-  <http://mu.semte.ch/vocabularies/ext/usesConceptScheme>   "false"^^xsd:boolean ;
-  <http://mu.semte.ch/vocabularies/ext/canHaveValidation>   <http://lblod.data.gift/vocabularies/forms/RequiredConstraint>, <http://lblod.data.gift/vocabularies/forms/MaxLength> ;
-  <http://mu.semte.ch/vocabularies/core/uuid>   "68ccebe5-84fb-4976-a9b1-28b1d3ef3687" ;
-  skos:inScheme   <http://lblod.data.gift/concept-schemes/c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6> ;
+  ext:displayType "defaultInput" ;
+  ext:usesConceptScheme "false"^^xsd:boolean ;
+  ext:canHaveValidation 
+      form:RequiredConstraint,
+      form:MaxLength,
+      form:ValidEmail,
+      form:ValidInteger,
+      form:UriConstraint,
+      form:ValidIBAN ;
+  core:uuid "68ccebe5-84fb-4976-a9b1-28b1d3ef3687" ;
+  skos:inScheme <http://lblod.data.gift/concept-schemes/c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6> ;
```

Comment:
I cannot change this to a `prefix` because than no options are shown in the dropdown. This is still the same `uri` as in the selector field in the `form.ttl` of the builder. Not sure why  I can't do this